packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,7 +1,22 @@
-##41.16.0 - January 8, 2018
+## 41.16.0 - January 8, 2018
 ### Added
-#### salesforce-vscode-apex
-* Update apex-jorje-lsp.jar to enable go to definition for type usage sites ([PR #247] (https://github.com/forcedotcom/salesforcedx-vscode/pull/247))
+#### All
+* Upgrade to TypeScript 2.6.2 ([PR #242] (https://github.com/forcedotcom/salesforcedx-vscode/pull/242))
+#### salesforcedx-vscode-apex
+* Enable Go To Definition for usages of classes and interfaces in class and interface declarations ([PR #247] (https://github.com/forcedotcom/salesforcedx-vscode/pull/247))
+#### scripts
+* Add script to enable testing with external dependencies ([PR #244] (https://github.com/forcedotcom/salesforcedx-vscode/pull/244))
+#### salesforcedx-apex-debugger
+* Add visual indication in the call stack telling the user what exception the user is currently paused on ([PR #240] (https://github.com/forcedotcom/salesforcedx-vscode/pull/240)
+#### salesforcedx-vscode-lwc
+* Add LWC Extension ([PR #241] (https://github.com/forcedotcom/salesforcedx-vscode/pull/241))
```

Comment:
Remove this. We will have a proper post about this later.

Reply:
Just the LWC part. I think the UI in GitHub is not very precise when it comes to the line numbers.